workflow/snakemake_rules/main_workflow.smk

```diff
@@ -503,13 +493,12 @@ rule build_align:
             --sequences {input.sequences} \
             --strip-prefixes {params.strain_prefixes:q} \
             --output /dev/stdout 2> {params.sanitize_log} \
-            | nextclade2 run \
+            | nextclade run \
             --jobs {threads} \
             --input-dataset {input.nextclade_dataset} \
             --output-tsv {output.nextclade_qc} \
             --output-fasta {output.alignment} \
-            --output-translations {params.output_translations} \
-            --output-insertions {output.insertions} 2>&1 | tee {log}
+            --output-translations {params.output_translations} 2>&1 | tee {log}
         """
 
 rule join_metadata_and_nextclade_qc:
```

Comment:
I was tempted to update this rule to use `augur merge` to join the metadata and Nextclade data. However, the custom `join-metadata-and-clades.py` script includes [calculations for `clock_deviation`](https://github.com/nextstrain/ncov/blob/66aa9be182df05f04acc5a2cbb4ad750b39eb0ec/scripts/join-metadata-and-clades.py#L125-L143) that I don't really want to touch on here. Will leave that for another time...